lib/log_printer.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:logger/logger.dart';
```

Comment:
Why not use
```
import 'dart:developer' as developer;
developer.log("")
```? 

https://api.flutter.dev/flutter/dart-developer/log.html

Reply:
Nvm. This `logger` looks pretty.

Reply:
Haha yes. That's the reason I chose this too :)